run_taxaHFEv2.R

```diff
@@ -15,99 +15,54 @@
+    -m --max_depth <int>              How many hierarchical levels should be allowed to compete [default: 1000]
+    -c --cor_level <float>            Initial pearson correlation filter [default: 0.95]
+    -n --ncores <int>                 Number of cpu cores to use [default: 2]
+    -d --disable_super_filter         Disable running of the super filter (final forest competition)
+    -w --write_old_files              Write individual level files and old HFE files
+    -W --write_flattened_tree         Write a compressed backup of the entire competed tree
+    -D --write_both_outputs           Write an output for pre and post super filter results, overridden by --disable_super_filter
+    --nperm <int>                     Number of RF permutations [default: 40]
+    --seed <numeric>                  Set a random numeric seed, default is to use system time
+
 Arguments:
     METADATA path to metadata input (txt | tsv | csv)
     DATA path to input file from hierarchical data (i.e. hData data) (txt | tsv | csv)
     OUTPUT output file name (csv)
 
 ' -> doc
 
-opt <- docopt::docopt(doc, version = 
-                        'taxaHFE.R v2.0\n\n')
-
-## load functions ==============================================================
-
-source("/scripts/utilities/tree.R")
-#source("/home/docker/tree.R")
-
-## arg tests ===================================================================
-# opt <- data.frame(subject_identifier = character(),
-#                   label = character(),
-#                   feature_type = character(),
-#                   ncores = numeric(),
-#                   cor_level = numeric(),
-#                   sample_fraction = numeric(),
-#                   abundance = numeric(),
-#                   prevalence = numeric(),
-#                   write_old_files = character(),
-#                   lowest_level = numeric(),
-#                   max_depth = numeric(),
-#                   nperm = numeric(),
-#                   seed = numeric(),
-#                   METADATA = character(),
-#                   DATA = character(),
-#                   OUTPUT = character())
-# opt <- opt %>% tibble::add_row(
-#   subject_identifier = "Sample",
-#   label = "Category",
-#   feature_type = "factor",
-#   write_old_files = "TRUE",
-#   abundance = 0.0001,
-#   prevalence = 0.01,
-#   sample_fraction = 1,
-#   cor_level = 0.95,
-#   lowest_level = 3,
-#   max_depth = 1000,
-#   ncores = 4,
-#   nperm = 10,
-#   seed = 42,
-#   METADATA = "/home/docker/example_inputs/metadata.txt",
-#   DATA = "/home/docker/example_inputs/microbiome_data.txt",
-#   OUTPUT = "/home/docker/example_inputs/output.csv"
-# )
+# these options will be converted to numeric by load_docopt
+numeric_options <- c("sample_fraction", "abundance", "prevalence", "lowest_level", "max_depth", "cor_level", "ncores", "nperm")
+# to use this code line-by-line in the Rstudio context, commandArgs can be overloaded to specify the desired flags
```

Comment:
this is the process to use flags in rstudio, might fit better in a development section in the readme?